### PR TITLE
LOR-83: updateReminder on view button click

### PR DESCRIPTION
**Pull request by Nishan212**

### CODE REVIEW

File: client/lib/reminders/hooks.tsx [hunk 1]
```diff
@@ -37,6 +44,45 @@ export const useReminders = (): RemindersReturn => {
...
 
+	const [updateReminderMutation] = useUpdateReminderMutation({
+		onError: error => {
+			toast({
+				title: "FAILED to update reminder status to 'viewed'",
+				description: error.message,
+				status: "error",
+				duration: 3000,
+				position: "top",
+				isClosable: true,
```
Comment (karan316) [inline client/lib/reminders/hooks.tsx:55]: this toast is not needed. console log it

File: client/lib/reminders/hooks.tsx [hunk 2]
```diff
@@ -37,6 +44,45 @@ export const useReminders = (): RemindersReturn => {
...
+			});
+		},
+		onCompleted: () => {
+			toast({
+				title: "SUCCESS",
+				description: "Reminder status Updated to 'viewed'",
+				status: "success",
+				duration: 3000,
+				position: "top",
+				isClosable: true,
```
Comment (karan316) [inline client/lib/reminders/hooks.tsx:65]: not needed. toast is only for the end-user. why would they care if the toast has been viewed or not?

File: client/lib/reminders/hooks.tsx [hunk 3]
```diff
@@ -37,6 +44,45 @@ export const useReminders = (): RemindersReturn => {
...
+			});
+			console.log(data);
+		} catch (error) {
+			console.error(error);
+		}
+	};
+
 	const [deleteReminderMutation] = useDeleteReminderMutation({
 		onError: error => {
 			toast({
```
Comment (karan316) [inline client/lib/reminders/hooks.tsx:88]: toast is not needed

**Review by karan316 — CHANGES_REQUESTED** [review]
toasts are not needed

File: client/lib/reminders/hooks.tsx (hunk 3 above)
Comment (Nishan212) [reply client/lib/reminders/hooks.tsx:88]: kidhar bro here?

**Review by karan316 — APPROVED** [review]

**Pull request merged**